bootable-jar/boot/src/main/java/org/wildfly/core/jar/boot/CleanupProcessor.java

```diff
@@ -34,6 +34,9 @@ public static void main(final String[] args) throws Exception {
         final Path installDir = Paths.get(args[0]);
         final int retries = Integer.parseInt(args[1]);
         final Path cleanupMarker = installDir.resolve("wildfly-cleanup-marker");
+        if (Files.notExists(cleanupMarker)) {
+            return;
+        }
```

Comment:
I'm not sure I follow this. Can you explain why we were seeing an issue if the file does not exist here?